packages/libllvm/llvm-config.in

```diff
@@ -40,68 +40,80 @@ Typical components:
   engine            Either a native JIT or a bitcode interpreter."
 }
 
-arch=@TERMUX_ARCH@
 version=@TERMUX_PKG_VERSION@
 prefix=@TERMUX_PREFIX@
 has_rtti=NO
 CPPFLAGS="-I${prefix}/include -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS"
-CFLAGS="${CPPFLAGS} ${CFLAGS}  -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS"
-CXXFLAGS="${CFLAGS} -fvisibility-inlines-hidden -Wcast-qual -Wnon-virtual-dtor -std=c++11 -fno-exceptions"
+CFLAGS="${CPPFLAGS} ${CFLAGS}"
+CXXFLAGS="${CFLAGS} -std=c++14 -fno-exceptions"
```

Comment:
The llvm-config binary doesn't have those three flags I removed, let me know if any should be kept.